interfaces/builtin/autopilot.go

```diff
@@ -0,0 +1,58 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
The name of the interface should match the filename. Since you have named this 'autopilot-introsection', you should use autopilot_introspection.go and autopilot_introsepection_test.go